framework/di/Container.php

```diff
@@ -349,6 +349,21 @@ public function getDefinitions()
     }
 
     /**
+     * Configures an object with the initial property values.
+     * @param object $object the object to be configured
+     * @param array $properties the property initial values given in terms of name-value pairs.
+     * @return object the object itself
```

Comment:
"the object itself" is correct in this context?

Reply:
Yes. You're passing an object and the same object is returned with properties set.